core/src/avm2/object/script_object.rs

```diff
@@ -128,6 +139,10 @@ impl<'gc> ScriptObjectData<'gc> {
         }
     }
 
+    pub fn install_slot(&mut self) {
```

Comment:
I believe this function is unused now, so we can remove it

Reply:
Good catch, done.